app/src/main/java/org/schabi/newpipe/streams/WebMReader.java

```diff
@@ -15,7 +15,7 @@
  */
 public class WebMReader {
 
-    //<editor-fold defaultState="collapsed" desc="constants">
+    //<editor-fold defaultstate="collapsed" desc="constants">

```

Comment:
and in this file, too

Reply:
¿what is wrong? the are "manual folding" comments [Code Folding - User Interface Specification](https://ui.netbeans.org/docs/ui/code_folding/cf_uispec.html#custom)

Reply:
If I understand it correctly, these comments are used by NetBeans to collapse certain code fragments. But they do not belong in the main repository code. Please remove them.
I can understand that the collapsed code helps you with coding. I'd suggest you to put these comments into a single commit on another branch which allows you to cerry pick the editor comments when working locally.

Reply:
Thanks :+1: